controller/model/identity_handlers.go

```diff
@@ -207,6 +207,15 @@ func (handler *IdentityHandler) Read(id string) (*Identity, error) {
 	return entity, nil
 }
 
+func (handler *IdentityHandler) ReadByName(name string) (*Identity, error) {
```

Comment:
It isn't immediately apparent why we need `ReadByName` for identity and edge service. I believe config types are referenced by name by REST API Clients, so that doesn't seem strange to me. Config type names are also relatively static.

I don't see the identity/service ones used and thus I don't understand why we need it. I want to ensure we are avoiding relying on names other than for initial lookups as names can be changed.

Reply:
These will be used by tunnel control channel handlers and the router identities code. Services are dialed by name, so we need to be able to look them up by name. Router identities will be linked to routers by name, so we look them up by name and when we're modifying names to keep them in sync, we can look them up by the old name.